src/main/java/org/bytedeco/javacpp/annotation/Adapter.java

```diff
@@ -10,22 +10,82 @@
 import org.bytedeco.javacpp.tools.Generator;
 
 /**
- * Specifies a C++ class to act as an adapter to convert the types of arguments.
- * Three such C++ classes made available by {@link Generator} are {@code StringAdapter},
- * {@code VectorAdapter}, and {@code SharedPtrAdapter} to bridge a few differences between
- * {@code std::string} and {@link String}; between {@code std::vector}, Java arrays of
- * primitive types, {@link Buffer}, and {@link Pointer}; and between {@code xyz::shared_ptr}
- * and {@link Pointer}. Adapter classes must define the following public members:
- * <ul>
- * <li> A constructor accepting 3 arguments (or more if {@link #argc()} > 1): a pointer, a size, and the owner pointer
- * <li> Another constructor that accepts a reference to the object of the other class
- * <li> A {@code static void deallocate(owner)} function
- * <li> Overloaded cast operators to both types, for references and pointers
- * <li> A {@code void assign(pointer, size, owner)} function
- * <li> A {@code size} member variable for arrays accessed via pointer
- * </ul>
- * To reduce further the amount of coding, this annotation can also be used on
- * other annotations, such as with {@link StdString}, {@link StdVector}, and {@link SharedPtr}.
```

Comment:
You've added more comments, but you've also removed information. Please add it back

Reply:
Do you mean the "to bridge difference..." part ? . Better now ?

Reply:
No, but just don't delete it is fine, no?

Reply:
I don't follow you, can you directly change the way you prefer ?
I added back "to bridge..." altering it a bit because the previous version omits adapters that have been added since.
If it's about the "to reduce further coding...", I think the "The helper annotations are shortcuts ...." gives the same information while being a bit more precise, but change as you like.